### PR TITLE
Improve help and add GIF to readme

### PR DESCRIPTION
**Pull request by KnVerey**
@kirs @ibawt @dalehamel 
Follow up for https://github.com/Shopify/kubernetes-deploy/pull/120 

- Adds a gif
- Improves --help 
- removes the more obscure option descriptions from the readme

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,11 +98,12 @@ This repo also includes related tools for [running tasks](#kubernetes-run) and [
 
 *Options:*
 
+Refer to `kubernetes-deploy --help` for the authoritative set of options.
+
+- `--template-dir=DIR`: Used to set the deploy directory. Set `$ENVIRONMENT` instead to use `config/deploy/$ENVIRONMENT`.
 - `--bindings=BINDINGS`: Makes additional variables available to your ERB templates. For example, `kubernetes-deploy my-app cluster1 --bindings=color=blue,size=large` will expose `color` and `size`.
-- `--verbose-log-prefix`: Prefixes each log line with `[#{context}][#{namespace}][#{severity}][#{datetime}]` instead of just `[#{severity}][#{datetime}]`. Useful if you're running multiple concurrent deploys in Shipit.
 - `--allow-protected-ns`: Enables you to deploy to `default`, `kube-system` or `kube-public` if you really know what you're doing. Must be used with `--no-prune` (pruning a protected namespace cannot be enabled).
```
Comment (kirs) [inline README.md]: Maybe we can drop this one too? I bet no too many people will use this tool to deploy their apps to `kube-system`

File: README.md [hunk 2]
```diff
@@ -18,7 +18,9 @@ Especially in a CI/CD environment, we need a clear, actionable pass/fail result
 
 This repo also includes related tools for [running tasks](#kubernetes-run) and [restarting deployments](#kubernetes-restart).
 
-![success](screenshots/success.png)
+
+
+![demo-deploy.gif](screenshots/deploy-demo.gif)
```
Comment (kirs) [inline README.md]: 😍 😍 😍

**Comment by ibawt** [comment]
YES a GIF.  but seriously I really like it.

File: README.md (hunk 2 above)
Comment (dalehamel) [inline README.md]: :heart:

File: README.md (hunk 1 above)
Comment (KnVerey) [inline README.md]: True 👍

**Pull request merged**